
<!DOCTYPE html>


<html lang="en" data-content_root="../" >

  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>6.2. Protein Modelling &#8212; Molecular Dynamics and Monte Carlo</title>
  
  
  
  <script data-cfasync="false">
    document.documentElement.dataset.mode = localStorage.getItem("mode") || "";
    document.documentElement.dataset.theme = localStorage.getItem("theme") || "";
  </script>
  
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/bootstrap.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
<link href="../_static/styles/pydata-sphinx-theme.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />

  
  <link href="../_static/vendor/fontawesome/6.5.2/css/all.min.css?digest=dfe6caa3a7d634c4db9b" rel="stylesheet" />
  <link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-solid-900.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-brands-400.woff2" />
<link rel="preload" as="font" type="font/woff2" crossorigin href="../_static/vendor/fontawesome/6.5.2/webfonts/fa-regular-400.woff2" />

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css?v=03e43079" />
    <link rel="stylesheet" type="text/css" href="../_static/styles/sphinx-book-theme.css?v=eba8b062" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css?v=13237357" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css?v=76b2166b" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.4510f1fc1dee50b3e5859aac5469c37c29e427902b24a333a5f9fcb2f0b3ac41.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css?v=4fa983c6" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-design.min.css?v=95c83b7e" />
  
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b" />
<link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b" />
  <script src="../_static/vendor/fontawesome/6.5.2/js/all.min.js?digest=dfe6caa3a7d634c4db9b"></script>

    <script src="../_static/documentation_options.js?v=9eb32ce0"></script>
    <script src="../_static/doctools.js?v=9a2dae69"></script>
    <script src="../_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="../_static/clipboard.min.js?v=a7894cd8"></script>
    <script src="../_static/copybutton.js?v=f281be69"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?v=887ef09a"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js?v=4a39c7ea"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script src="../_static/design-tabs.js?v=f930bc37"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script async="async" src="../_static/sphinx-thebe.js?v=c100c467"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown';</script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"; const thebe_selector = ".thebe,.cell"; const thebe_selector_input = "pre"; const thebe_selector_output = ".output, .cell_output"</script>
    <script>window.MathJax = {"TeX": {"Macros": {"ket": ["\\lvert #1 \\rangle", 1], "bra": ["\\langle #1 \\lvert", 1], "Ket": ["\\lvert #1 \\rangle", 1], "Bra": ["\\langle #1 \\lvert", 1], "braket": ["\\langle #1 \\lvert #2 \\rangle", 2], "hop": "\\hat{\\mathcal{H}}", "unit": "\\mathbbm{1}"}}, "options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <script>DOCUMENTATION_OPTIONS.pagename = 'Ex6/proteinmodelling';</script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="6.3. Practical: TRP Cage using OpenMM" href="TRP_Cage.html" />
    <link rel="prev" title="6.1. Theory" href="theory.html" />
  <meta name="viewport" content="width=device-width, initial-scale=1"/>
  <meta name="docsearch:language" content="en"/>
  </head>
  
  
  <body data-bs-spy="scroll" data-bs-target=".bd-toc-nav" data-offset="180" data-bs-root-margin="0px 0px -60%" data-default-mode="">

  
  
  <div id="pst-skip-link" class="skip-link d-print-none"><a href="#main-content">Skip to main content</a></div>
  
  <div id="pst-scroll-pixel-helper"></div>
  
  <button type="button" class="btn rounded-pill" id="pst-back-to-top">
    <i class="fa-solid fa-arrow-up"></i>Back to top</button>

  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-primary-sidebar-checkbox"/>
  <label class="overlay overlay-primary" for="pst-primary-sidebar-checkbox"></label>
  
  <input type="checkbox"
          class="sidebar-toggle"
          id="pst-secondary-sidebar-checkbox"/>
  <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
  
  <div class="search-button__wrapper">
    <div class="search-button__overlay"></div>
    <div class="search-button__search-container">
<form class="bd-search d-flex align-items-center"
      action="../search.html"
      method="get">
  <i class="fa-solid fa-magnifying-glass"></i>
  <input type="search"
         class="form-control"
         name="q"
         id="search-input"
         placeholder="Search this book..."
         aria-label="Search this book..."
         autocomplete="off"
         autocorrect="off"
         autocapitalize="off"
         spellcheck="false"/>
  <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd>K</kbd></span>
</form></div>
  </div>

  <div class="pst-async-banner-revealer d-none">
  <aside id="bd-header-version-warning" class="d-none d-print-none" aria-label="Version warning"></aside>
</div>

  
    <header class="bd-header navbar navbar-expand-lg bd-navbar d-print-none">
    </header>
  

  <div class="bd-container">
    <div class="bd-container__inner bd-page-width">
      
      
      
      <div class="bd-sidebar-primary bd-sidebar">
        

  
  <div class="sidebar-header-items sidebar-primary__section">
    
    
    
    
  </div>
  
    <div class="sidebar-primary-items__start sidebar-primary__section">
        <div class="sidebar-primary-item">

  
    
  

<a class="navbar-brand logo" href="../intro.html">
  
  
  
  
  
  
    <p class="title logo__title">Molecular Dynamics and Monte Carlo</p>
  
</a></div>
        <div class="sidebar-primary-item">

 <script>
 document.write(`
   <button class="btn search-button-field search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass"></i>
    <span class="search-button__default-text">Search</span>
    <span class="search-button__kbd-shortcut"><kbd class="kbd-shortcut__modifier">Ctrl</kbd>+<kbd class="kbd-shortcut__modifier">K</kbd></span>
   </button>
 `);
 </script></div>
        <div class="sidebar-primary-item"><nav class="bd-links bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item navbar-nav active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../intro.html">
                    Molecular Dynamics and Monte Carlo
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading"><span class="caption-text">Exercises</span></p>
<ul class="current nav bd-sidenav">
<li class="toctree-l1 has-children"><a class="reference internal" href="../Ex1/MDMC_Ex1.html">1. Basic Concepts of Molecular Dynamics and Monte Carlo Simulations</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="../Ex1/Ex1_Theory.html">1.1. From Quantum to Classical Mechanics: The Example of Forces</a></li>

<li class="toctree-l2"><a class="reference internal" href="../Ex1/MonteCarloPI.html">1.3. Monte Carlo <span class="math notranslate nohighlight">\(\pi\)</span></a></li>
<li class="toctree-l2"><a class="reference internal" href="../Ex1/prng.html">1.4. Appendix: Random and Pseudo-random Numbers</a></li>
</ul>
</details></li>
<li class="toctree-l1 has-children"><a class="reference internal" href="../Ex2/MDMC_ex2.html">2. Statistical Thermodynamics and the Boltzmann Distribution</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="../Ex2/Ex2_Theory.html">2.1. Statistical Thermodynamics and the Boltzmann Distribution</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Ex2/Boltzmann.html">2.2. Boltzmann Distribution Harmonic Oscillator</a></li>
</ul>
</details></li>
<li class="toctree-l1 has-children"><a class="reference internal" href="../Ex3/MDMC_ex3.html">3. Monte Carlo Simulations and the Photon Gas</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="../Ex3/Ex3_Theory.html">3.1. Theory</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Ex3/PhotonGas.html">3.2. The Photon Gas</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Ex3/LJ_NVT.html">3.3. The Lennard-Jones Potential</a></li>
</ul>
</details></li>
<li class="toctree-l1 has-children"><a class="reference internal" href="../Ex4/MDMC_ex4.html">4. Introduction to Molecular Dynamics</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="../Ex4/Theory.html">4.1. Theory</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Ex4/Exercise.html">4.2. MD simulation from scratch</a></li>

</ul>
</details></li>
<li class="toctree-l1 has-children"><a class="reference internal" href="../Ex5/MDMC_ex5.html">5. Classical Molecular Dynamics (MD) Simulations</a><details><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul>
<li class="toctree-l2"><a class="reference internal" href="../Ex5/theory.html">5.1. Theory</a></li>
<li class="toctree-l2"><a class="reference internal" href="../Ex5/Exercise.html">5.2. ToyMD</a></li>

</ul>
</details></li>
<li class="toctree-l1 current active has-children"><a class="reference internal" href="MDMC_ex6.html">6. Properties from Molecular Dynamics</a><details open="open"><summary><span class="toctree-toggle" role="presentation"><i class="fa-solid fa-chevron-down"></i></span></summary><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="theory.html">6.1. Theory</a></li>
<li class="toctree-l2 current active"><a class="current reference internal" href="#">6.2. Protein Modelling</a></li>
<li class="toctree-l2"><a class="reference internal" href="TRP_Cage.html">6.3. Practical: TRP Cage using OpenMM</a></li>

<li class="toctree-l2"><a class="reference internal" href="hydrogenbonds.html">6.5. TrpCage Analysis</a></li>
</ul>
</details></li>
</ul>

    </div>
</nav></div>
    </div>
  
  
  <div class="sidebar-primary-items__end sidebar-primary__section">
  </div>
  
  <div id="rtd-footer-container"></div>


      </div>
      
      <main id="main-content" class="bd-main" role="main">
        
        

<div class="sbt-scroll-pixel-helper"></div>

          <div class="bd-content">
            <div class="bd-article-container">
              
              <div class="bd-header-article d-print-none">
<div class="header-article-items header-article__inner">
  
    <div class="header-article-items__start">
      
        <div class="header-article-item"><button class="sidebar-toggle primary-toggle btn btn-sm" title="Toggle primary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
  <span class="fa-solid fa-bars"></span>
</button></div>
      
    </div>
  
  
    <div class="header-article-items__end">
      
        <div class="header-article-item">

<div class="article-header-buttons">





<div class="dropdown dropdown-source-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Source repositories">
    <i class="fab fa-github"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="https://github.com/lcbc-epfl/mdmc-public" target="_blank"
   class="btn btn-sm btn-source-repository-button dropdown-item"
   title="Source repository"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="btn__text-container">Repository</span>
</a>
</li>
      
      
      
      
      <li><a href="https://github.com/lcbc-epfl/mdmc-public/issues/new?title=Issue%20on%20page%20%2FEx6/proteinmodelling.html&body=Your%20issue%20content%20here." target="_blank"
   class="btn btn-sm btn-source-issues-button dropdown-item"
   title="Open an issue"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="btn__text-container">Open issue</span>
</a>
</li>
      
  </ul>
</div>






<div class="dropdown dropdown-download-buttons">
  <button class="btn dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false" aria-label="Download this page">
    <i class="fas fa-download"></i>
  </button>
  <ul class="dropdown-menu">
      
      
      
      <li><a href="../_sources/Ex6/proteinmodelling.md" target="_blank"
   class="btn btn-sm btn-download-source-button dropdown-item"
   title="Download source file"
   data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="btn__text-container">.md</span>
</a>
</li>
      
      
      
      
      <li>
<button onclick="window.print()"
  class="btn btn-sm btn-download-pdf-button dropdown-item"
  title="Print to PDF"
  data-bs-placement="left" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="btn__text-container">.pdf</span>
</button>
</li>
      
  </ul>
</div>




<button onclick="toggleFullScreen()"
  class="btn btn-sm btn-fullscreen-button"
  title="Fullscreen mode"
  data-bs-placement="bottom" data-bs-toggle="tooltip"
>
  

<span class="btn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>



<script>
document.write(`
  <button class="btn btn-sm nav-link pst-navbar-icon theme-switch-button" title="light/dark" aria-label="light/dark" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="theme-switch fa-solid fa-sun fa-lg" data-mode="light"></i>
    <i class="theme-switch fa-solid fa-moon fa-lg" data-mode="dark"></i>
    <i class="theme-switch fa-solid fa-circle-half-stroke fa-lg" data-mode="auto"></i>
  </button>
`);
</script>


<script>
document.write(`
  <button class="btn btn-sm pst-navbar-icon search-button search-button__button" title="Search" aria-label="Search" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <i class="fa-solid fa-magnifying-glass fa-lg"></i>
  </button>
`);
</script>
<button class="sidebar-toggle secondary-toggle btn btn-sm" title="Toggle secondary sidebar" data-bs-placement="bottom" data-bs-toggle="tooltip">
    <span class="fa-solid fa-list"></span>
</button>
</div></div>
      
    </div>
  
</div>
</div>
              
              

<div id="jb-print-docs-body" class="onlyprint">
    <h1>Protein Modelling</h1>
    <!-- Table of contents -->
    <div id="print-main-content">
        <div id="jb-print-toc">
            
            <div>
                <h2> Contents </h2>
            </div>
            <nav aria-label="Page">
                <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#molecular-dynamics-simulations-in-biochemistry">6.2.1. Molecular Dynamics Simulations in Biochemistry</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#amino-acids-and-proteins">6.2.2. Amino Acids and Proteins</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#amino-acids">6.2.2.1. Amino Acids</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#proteins">6.2.2.2. Proteins</a><ul class="nav section-nav flex-column">
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#proteins-form-and-function">6.2.2.2.1. Proteins: Form and Function</a></li>
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#alpha-helices-and-beta-sheets">6.2.2.2.2. <span class="math notranslate nohighlight">\(\alpha\)</span>-helices and <span class="math notranslate nohighlight">\(\beta\)</span>-sheets</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#protein-assemblies">6.2.3. Protein Assemblies</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#secondary-structures">6.2.3.1. Secondary Structures</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#tertiary-structures">6.2.3.2. Tertiary Structures</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#quaternary-structures">6.2.3.3. Quaternary Structures</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#protein-folding">6.2.4. Protein Folding</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#computational-methods">6.2.5. Computational methods</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein">6.2.5.1. Analysis of the Folding Dynamics of the Trp-cage Miniprotein</a></li>
</ul>
</li>
</ul>
            </nav>
        </div>
    </div>
</div>

              
                
<div id="searchbox"></div>
                <article class="bd-article">
                  
  <section class="tex2jax_ignore mathjax_ignore" id="protein-modelling">
<h1><span class="section-number">6.2. </span>Protein Modelling<a class="headerlink" href="#protein-modelling" title="Link to this heading">#</a></h1>
<section id="molecular-dynamics-simulations-in-biochemistry">
<h2><span class="section-number">6.2.1. </span>Molecular Dynamics Simulations in Biochemistry<a class="headerlink" href="#molecular-dynamics-simulations-in-biochemistry" title="Link to this heading">#</a></h2>
<p>Individual cellular processes are complex; involving a plethora of
reactants, reaction pathways, intermediary states and biological
catalysts. Such processes are often very large (<span class="math notranslate nohighlight">\(&gt;1,000,000\)</span> atoms), and
with modern computing power, treating such large systems at a full <em>ab
initio</em> level is not appropriate. Understanding these reaction pathways
in view of developing drugs or treatments for diseases is often an
immense undertaking. For example, the structural determination of
proteins by experiment requires that these proteins are synthesised,
crystallised and purified. One may, however, encounter difficult
transfection, suffer from low yields or high losses during purification:
all of which lead to very high monetary cost. Crystallisation techniques
often do not capture the protein of interest in its active state, and
cannot isolate some proteins at all, hence may not be of much benefit to
perform.</p>
<p>Computational modelling is generally a useful tool in cases where
mechanisms or structures are poorly understood. Additionally when using
molecular dynamics, processes can be captured explicitly, thus analysis
can be performed in visual and mathematical detail, in an environment in
which the protein is active.</p>
</section>
<section id="amino-acids-and-proteins">
<h2><span class="section-number">6.2.2. </span>Amino Acids and Proteins<a class="headerlink" href="#amino-acids-and-proteins" title="Link to this heading">#</a></h2>
<section id="amino-acids">
<h3><span class="section-number">6.2.2.1. </span>Amino Acids<a class="headerlink" href="#amino-acids" title="Link to this heading">#</a></h3>
<p>Amino acids are the structural units that make up proteins. They join
together via dehydration (<a class="reference internal" href="#dehydration-reaction"><span class="std std-numref">Fig. 6.1</span></a>) to form short polymer chains named peptides,
or longer chains called polypeptides or proteins. These polypeditic
chains are unbranched, with each amino acid within the chain attached to
two neighbouring amino acids. There are a total of 23 amino acids that
are naturally incorporated into polypeptides, termed proteinogenic amino
acids. Of these, 21 are encoded by the universal genetic code, while the
remaining two, selenocysteine and pyrrolysine, are incorporated into
polypeptides by unique biological mechanisms.</p>
<figure class="align-default" id="dehydration-reaction">
<img alt="../_images/dehydration_reaction.png" src="../_images/dehydration_reaction.png" />
<figcaption>
<p><span class="caption-number">Fig. 6.1 </span><span class="caption-text">Dehydration of two amino acids to yield a dipeptide. Each amino acid
differs in the <strong>R</strong> group, thus polypeptide chains have a preserved and
regular backbone. Public domain <a class="reference external" href="https://commons.wikimedia.org/wiki/File:AminoacidCondensation.svg">V8rik</a></span><a class="headerlink" href="#dehydration-reaction" title="Link to this image">#</a></p>
</figcaption>
</figure>
</section>
<section id="proteins">
<h3><span class="section-number">6.2.2.2. </span>Proteins<a class="headerlink" href="#proteins" title="Link to this heading">#</a></h3>
<p>Proteins are large macromolecules consisting of one or more polypeptide
chains. They differ from one another primarily in their sequence of
amino acids, which is dictated by the genetic coding responsible for
that particular protein. The amino acid sequence results in a folded
protein with a specific structure that determines its activity for a
particular process. Proteins perform the majority of the vast array of
functions required for life within living organisms, including:
metabolic catalysis, DNA replication and signal transduction.</p>
<section id="proteins-form-and-function">
<h4><span class="section-number">6.2.2.2.1. </span>Proteins: Form and Function<a class="headerlink" href="#proteins-form-and-function" title="Link to this heading">#</a></h4>
<p>The activity and task of a protein, in large part, is determined by its
3D structural characteristics. This is defined by the sequence of amino
acids in the polypeptide chains that constitute the protein, which, in
turn, is defined by the sequence of genetic code on the encoding mRNA
strand.</p>
</section>
<section id="alpha-helices-and-beta-sheets">
<span id="alpha-helix"></span><h4><span class="section-number">6.2.2.2.2. </span><span class="math notranslate nohighlight">\(\alpha\)</span>-helices and <span class="math notranslate nohighlight">\(\beta\)</span>-sheets<a class="headerlink" href="#alpha-helices-and-beta-sheets" title="Link to this heading">#</a></h4>
<p>Polypeptide chains are not usually linear, in fact, they can adopt very
specific conformations which are vitally important to the structural
characteristics of proteins. The most common conformations are the
<span class="math notranslate nohighlight">\(\alpha\)</span>-helix and <span class="math notranslate nohighlight">\(\beta\)</span>-sheet (<a class="reference internal" href="#polypeptide-structures"><span class="std std-numref">Fig. 6.2</span></a>), however, there are variants which
differ in their symmetry. The <span class="math notranslate nohighlight">\(\alpha\)</span>-helix conformation results when
every backbone <span class="math notranslate nohighlight">\(N-H\)</span> group donates a hydrogen bond to the backbone <span class="math notranslate nohighlight">\(C=O\)</span>
of the amino acid four residues earlier (<span class="math notranslate nohighlight">\(i+4 \rightarrow i\)</span> hydrogen
bonding). Conversely, <span class="math notranslate nohighlight">\(\beta\)</span>-sheets consist of <span class="math notranslate nohighlight">\(\beta\)</span>-strands (linear
polypeptides) connected laterally by at least two or three backbone
hydrogen bonds, forming a generally twisted pleated sheet.</p>
<p>There are multiple larger-scale structures in which a number of
<span class="math notranslate nohighlight">\(\alpha\)</span>-helical or <span class="math notranslate nohighlight">\(\beta\)</span>-sheets assemble into complicated motifs.
These include: coiled-coils, <span class="math notranslate nohighlight">\(\beta\)</span>-hairpin, Greek key motif,
<span class="math notranslate nohighlight">\(\beta\)</span>-<span class="math notranslate nohighlight">\(\alpha\)</span>-<span class="math notranslate nohighlight">\(\beta\)</span> motif, <span class="math notranslate nohighlight">\(\beta\)</span>-meander motif and <span class="math notranslate nohighlight">\(\Psi\)</span>-loop
motif.</p>
 <script src="https://3Dmol.org/build/3Dmol-min.js" async></script>     
<div style="height: 400px; width: 100%; position: relative;" class='viewer_3Dmoljs' data-href='alpha_helix.pdb' data-backgroundcolor='0xffffff' ddata-select1='chain:A' data-style1='cartoon' data-select2='chain:A' data-style2='cartoon;stick'></div> <div style="height: 400px; width: 100%; position: relative;" class='viewer_3Dmoljs' data-href='beta_sheet.pdb' data-backgroundcolor='0xffffff' data-select1='chain:A' data-style1='cartoon' data-select2='chain:A' data-style2='cartoon;stick'></div>       
<figure class="align-default" id="polypeptide-structures">
<a class="reference internal image-reference" href="../_images/protein-structures2.png"><img alt="../_images/protein-structures2.png" src="../_images/protein-structures2.png" style="height: 1px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 6.2 </span><span class="caption-text">Hydrogen-bonding is the primary driving force of the formation of <span class="math notranslate nohighlight">\(\alpha\)</span>-helices and <span class="math notranslate nohighlight">\(\beta\)</span>-sheets. This in turn depends on the type and sequence of amino acids involved in the polypeptide. Top alpha helix from the GB1 protein, bottom beta hairpin from the GB1 protein containing a beta sheet connected with a short loop.</span><a class="headerlink" href="#polypeptide-structures" title="Link to this image">#</a></p>
</figcaption>
</figure>
</section>
</section>
</section>
<section id="protein-assemblies">
<h2><span class="section-number">6.2.3. </span>Protein Assemblies<a class="headerlink" href="#protein-assemblies" title="Link to this heading">#</a></h2>
<p>There are three common structures involving proteins: secondary,
tertiary and quaternary structures (<a class="reference internal" href="#protein-structures"><span class="std std-numref">Fig. 6.3</span></a>). The study of these structures is vital
to the understanding of diseases, metabolism and other biological
processes.</p>
<section id="secondary-structures">
<h3><span class="section-number">6.2.3.1. </span>Secondary Structures<a class="headerlink" href="#secondary-structures" title="Link to this heading">#</a></h3>
<p>Secondary structures evolve from the interaction behaviour of hydrogen
bonding formed between the amine hydrogen and carbonyl oxygen atoms
contained in the backbone peptide bonds of the protein. There are a
number of secondary structures observed in nature, the two most common
being the <span class="math notranslate nohighlight">\(\alpha\)</span>-helix and <span class="math notranslate nohighlight">\(\beta\)</span>-sheet described in <a class="reference internal" href="#alpha-helix"><span class="std std-numref">Section 6.2.2.2.2</span></a></p>
</section>
<section id="tertiary-structures">
<h3><span class="section-number">6.2.3.2. </span>Tertiary Structures<a class="headerlink" href="#tertiary-structures" title="Link to this heading">#</a></h3>
<p>Tertiary structures are proteins which have a single polypeptide chain,
with one or more protein secondary structures.</p>
</section>
<section id="quaternary-structures">
<h3><span class="section-number">6.2.3.3. </span>Quaternary Structures<a class="headerlink" href="#quaternary-structures" title="Link to this heading">#</a></h3>
<p>Quaternary structures involve the folding of multiple folded protein or
coiling protein molecules in a multi-subunit complex.</p>
<figure class="align-default" id="protein-structures">
<a class="reference internal image-reference" href="../_images/protein_structure.jpg"><img alt="../_images/protein_structure.jpg" src="../_images/protein_structure.jpg" style="height: 400px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 6.3 </span><span class="caption-text">Depiction of the components of quaternary
structures. (CC by SA 3.0 <a class="reference external" href="http://cnx.org/content/col11496/1.6/">
OpenStax College - Anatomy &amp; Physiology,</a>)</span><a class="headerlink" href="#protein-structures" title="Link to this image">#</a></p>
</figcaption>
</figure>
</section>
</section>
<section id="protein-folding">
<h2><span class="section-number">6.2.4. </span>Protein Folding<a class="headerlink" href="#protein-folding" title="Link to this heading">#</a></h2>
<p>Protein folding is the process by which a protein assumes its functional
shape or conformation. Each protein exists as an unfolded polypeptide
chain when translated from a sequence of mRNA to a linear chain of amino
acids (Figure <a class="reference internal" href="#id1"><span class="std std-numref">Fig. 6.4</span></a>). Early folding already occurs in the exit tunnel of the ribosome (such as formation of alpha helices). After exiting the polypeptide
chain then further folds up in a complex process than can involve multiple intermediary state which sometimes require chaperone proteins to fold the protein into the global energy minimum. This free energy minimum exists because displacement of ordered solvent around the hydrophobic core into bulk solvent is favorable (so called hydrophobic collapse, <a class="reference internal" href="#id1"><span class="std std-numref">Fig. 6.4</span></a>).</p>
<figure class="align-default" id="id1">
<a class="reference internal image-reference" href="../_images/Protein_folding_schematic.png"><img alt="../_images/Protein_folding_schematic.png" src="../_images/Protein_folding_schematic.png" style="height: 400px;" /></a>
<figcaption>
<p><span class="caption-number">Fig. 6.4 </span><span class="caption-text">An example of a protein primary chain folding into a complex tertiary
structure by hydrophobic collapse. Filled black dots are hydrophobic amino acids. Public Domain <a class="reference external" href="https://commons.wikimedia.org/wiki/File:Protein_folding_schematic.png">Tomixdf</a></span><a class="headerlink" href="#id1" title="Link to this image">#</a></p>
</figcaption>
</figure>
</section>
<section id="computational-methods">
<span id="id2"></span><h2><span class="section-number">6.2.5. </span>Computational methods<a class="headerlink" href="#computational-methods" title="Link to this heading">#</a></h2>
<section id="analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein">
<h3><span class="section-number">6.2.5.1. </span>Analysis of the Folding Dynamics of the Trp-cage Miniprotein<a class="headerlink" href="#analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein" title="Link to this heading">#</a></h3>
<p>The Trp-cage miniprotein is a designed and stable polypeptide that
behaves like larger globular proteins. It is currently the smallest
protein to display two state folding properties which is easy to analyze in the laboratory e.g via thermal or chemical denaturation. The small size of this
protein makes it an ideal candidate for computational folding
simulations, thus it is frequently used to benchmark new computational
techniques and as a model system to compare folding morphologies in
larger systems. Simulating the folding dynamics of this protein with
explicitly defined water molecules is both a lengthy and computationally
costly process <a class="reference internal" href="#trpcagekinetics"><span class="std std-numref">Fig. 6.5</span></a>. Instead, we make use of an implicit
solvent model to reduce computational cost and still include, in
approximate detail, the effects of water solvation. This is inspired by work of Simmerling et. al.<span id="id3">[<a class="reference internal" href="../intro.html#id4" title="Carlos Simmerling, Bentley Strockbine, and Adrian E. Roitberg. All-atom structure prediction and folding simulations of a stable protein. Journal of the American Chemical Society, 124(38):11258-11259, 2002. doi:10.1021/ja0273851.">SSR02</a>]</span> in 2004.</p>
<figure class="align-default" id="trpcagekinetics">
<img alt="../_images/trpc.png" src="../_images/trpc.png" />
<figcaption>
<p><span class="caption-number">Fig. 6.5 </span><span class="caption-text">A computationally determined kinetic model of Trp-cage showing different states in the folding process and their interconversion rates. Yellow unfolded protein, red folded state, gray molten globule (i.e partially folded states)  <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000452">Marinelli et. al.</a> CC by SA</span><a class="headerlink" href="#trpcagekinetics" title="Link to this image">#</a></p>
</figcaption>
</figure>
<figure class="align-default" id="startingtrpstructure">
<img alt="../_images/startingStructure2.png" src="../_images/startingStructure2.png" />
<figcaption>
<p><span class="caption-number">Fig. 6.6 </span><span class="caption-text">Trp-cage starting structure in which cyan represents carbon, red is oxygen, white is hydrogen and blue is nitrogen. Built using the xLeap program within the AmberTools package.</span><a class="headerlink" href="#startingtrpstructure" title="Link to this image">#</a></p>
</figcaption>
</figure>
</section>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            name: "python3",
            path: "./Ex6"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

                </article>
              

              
              
              
              
                <footer class="prev-next-footer d-print-none">
                  
<div class="prev-next-area">
    <a class="left-prev"
       href="theory.html"
       title="previous page">
      <i class="fa-solid fa-angle-left"></i>
      <div class="prev-next-info">
        <p class="prev-next-subtitle">previous</p>
        <p class="prev-next-title"><span class="section-number">6.1. </span>Theory</p>
      </div>
    </a>
    <a class="right-next"
       href="TRP_Cage.html"
       title="next page">
      <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title"><span class="section-number">6.3. </span>Practical: TRP Cage using OpenMM</p>
      </div>
      <i class="fa-solid fa-angle-right"></i>
    </a>
</div>
                </footer>
              
            </div>
            
            
              
                <div class="bd-sidebar-secondary bd-toc"><div class="sidebar-secondary-items sidebar-secondary__inner">


  <div class="sidebar-secondary-item">
  <div class="page-toc tocsection onthispage">
    <i class="fa-solid fa-list"></i> Contents
  </div>
  <nav class="bd-toc-nav page-toc">
    <ul class="visible nav section-nav flex-column">
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#molecular-dynamics-simulations-in-biochemistry">6.2.1. Molecular Dynamics Simulations in Biochemistry</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#amino-acids-and-proteins">6.2.2. Amino Acids and Proteins</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#amino-acids">6.2.2.1. Amino Acids</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#proteins">6.2.2.2. Proteins</a><ul class="nav section-nav flex-column">
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#proteins-form-and-function">6.2.2.2.1. Proteins: Form and Function</a></li>
<li class="toc-h4 nav-item toc-entry"><a class="reference internal nav-link" href="#alpha-helices-and-beta-sheets">6.2.2.2.2. <span class="math notranslate nohighlight">\(\alpha\)</span>-helices and <span class="math notranslate nohighlight">\(\beta\)</span>-sheets</a></li>
</ul>
</li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#protein-assemblies">6.2.3. Protein Assemblies</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#secondary-structures">6.2.3.1. Secondary Structures</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#tertiary-structures">6.2.3.2. Tertiary Structures</a></li>
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#quaternary-structures">6.2.3.3. Quaternary Structures</a></li>
</ul>
</li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#protein-folding">6.2.4. Protein Folding</a></li>
<li class="toc-h2 nav-item toc-entry"><a class="reference internal nav-link" href="#computational-methods">6.2.5. Computational methods</a><ul class="nav section-nav flex-column">
<li class="toc-h3 nav-item toc-entry"><a class="reference internal nav-link" href="#analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein">6.2.5.1. Analysis of the Folding Dynamics of the Trp-cage Miniprotein</a></li>
</ul>
</li>
</ul>
  </nav></div>

</div></div>
              
            
          </div>
          <footer class="bd-footer-content">
            
<div class="bd-footer-content__inner container">
  
  <div class="footer-item">
    
<p class="component-author">
By Laboratory of Computational Biochemistry and Chemistry - Ursula Roethlisberger
</p>

  </div>
  
  <div class="footer-item">
    

  </div>
  
  <div class="footer-item">
    
  </div>
  
  <div class="footer-item">
    
  </div>
  
</div>
          </footer>
        

      </main>
    </div>
  </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/bootstrap.js?digest=dfe6caa3a7d634c4db9b"></script>
<script src="../_static/scripts/pydata-sphinx-theme.js?digest=dfe6caa3a7d634c4db9b"></script>

  <footer class="bd-footer">
  </footer>
  </body>
</html>