
<!DOCTYPE html>

<html lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>6.3. Protein Modelling &#8212; Molecular Dynamics and Monte Carlo</title>
    
  <link href="../_static/css/theme.css" rel="stylesheet">
  <link href="../_static/css/index.ff1ffe594081f20da1ef19478df9384b.css" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    
      

    
    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-book-theme.css?digest=c3fdc42140077d1ad13ad2f1588a4309" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/custom.css" />
    <link rel="stylesheet" type="text/css" href="../_static/panels-main.c949a650a448cc0ae9fd3441c0e17fb0.css" />
    <link rel="stylesheet" type="text/css" href="../_static/panels-variables.06eb56fa6e07937060861dad626602ad.css" />
    
  <link rel="preload" as="script" href="../_static/js/index.be7d3bbb2ef33a8344ce.js">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../_static/sphinx-book-theme.d59cb220de22ca1c485ebbdc042f0030.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <script>window.MathJax = {"TeX": {"Macros": {"ket": ["\\lvert #1 \\rangle", 1], "bra": ["\\langle #1 \\lvert", 1], "Ket": ["\\lvert #1 \\rangle", 1], "Bra": ["\\langle #1 \\lvert", 1], "braket": ["\\langle #1 \\lvert #2 \\rangle", 2], "hop": "\\hat{\\mathcal{H}}", "unit": "\\mathbbm{1}"}}, "options": {"processHtmlClass": "tex2jax_process|mathjax_process|math|output_area"}}</script>
    <script defer="defer" src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="6.4. Practical: TRP Cage using OpenMM" href="TRP_Cage_OpenMM_colab.html" />
    <link rel="prev" title="6.1. Theory" href="theory.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="en">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="80">
    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<div class="col-12 col-md-3 bd-sidebar site-navigation show" id="site-navigation">
    
        <div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Molecular Dynamics and Monte Carlo</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        <ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../intro.html">
   Molecular Dynamics and Monte Carlo
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Exercises
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../Ex1/MDMC_ex1.html">
   1. Basic Concepts of Molecular Dynamics and Monte Carlo Simulations
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-1" name="toctree-checkbox-1" type="checkbox"/>
  <label for="toctree-checkbox-1">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex1/Ex1_Theory.html">
     1.1. From Quantum to Classical Mechanics: The Example of Forces
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex1/MonteCarloPI.html">
     1.3. Monte Carlo
     <span class="math notranslate nohighlight">
      \(\pi\)
     </span>
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex1/prng.html">
     1.4. Appendix: Random and Pseudo-random Numbers
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../Ex2/MDMC_ex2.html">
   2. Statistical Thermodynamics and the Boltzmann Distribution
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-2" name="toctree-checkbox-2" type="checkbox"/>
  <label for="toctree-checkbox-2">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex2/Ex2_Theory.html">
     2.1. Statistical Thermodynamics and the Boltzmann Distribution
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex2/Boltzmann.html">
     2.2. Boltzmann Distribution Harmonic Oscillator
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../Ex3/MDMC_ex3.html">
   3. Monte Carlo Simulations and the Photon Gas
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-3" name="toctree-checkbox-3" type="checkbox"/>
  <label for="toctree-checkbox-3">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex3/Ex3_Theory.html">
     3.1. Theory
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex3/PhotonGas.html">
     3.2. The Photon Gas
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex3/LJ_NVT.html">
     3.3. The Lennard-Jones Potential
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../Ex4/MDMC_ex4.html">
   4. Introduction to Molecular Dynamics
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-4" name="toctree-checkbox-4" type="checkbox"/>
  <label for="toctree-checkbox-4">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex4/Theory.html">
     4.1. Theory
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex4/Exercise.html">
     4.2. MD simulation from scratch
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 has-children">
  <a class="reference internal" href="../Ex5/MDMC_ex5.html">
   5. Classical Molecular Dynamics (MD) Simulations
  </a>
  <input class="toctree-checkbox" id="toctree-checkbox-5" name="toctree-checkbox-5" type="checkbox"/>
  <label for="toctree-checkbox-5">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex5/theory.html">
     5.1. Theory
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="../Ex5/Exercise.html">
     5.2. ToyMD
    </a>
   </li>
  </ul>
 </li>
 <li class="toctree-l1 current active has-children">
  <a class="reference internal" href="MDMC_ex6.html">
   6. Properties from Molecular Dynamics
  </a>
  <input checked="" class="toctree-checkbox" id="toctree-checkbox-6" name="toctree-checkbox-6" type="checkbox"/>
  <label for="toctree-checkbox-6">
   <i class="fas fa-chevron-down">
   </i>
  </label>
  <ul class="current">
   <li class="toctree-l2">
    <a class="reference internal" href="theory.html">
     6.1. Theory
    </a>
   </li>
   <li class="toctree-l2 current active">
    <a class="current reference internal" href="#">
     6.3. Protein Modelling
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="TRP_Cage_OpenMM_colab.html">
     6.4. Practical: TRP Cage using OpenMM
    </a>
   </li>
   <li class="toctree-l2">
    <a class="reference internal" href="hydrogenbonds.html">
     6.5. TrpCage Analysis
    </a>
   </li>
  </ul>
 </li>
</ul>

    </div>
</nav> <!-- To handle the deprecated key -->

<div class="navbar_extra_footer">
  Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
</div>

</div>


          


          
<main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
    
    <div class="topbar container-xl fixed-top">
    <div class="topbar-contents row">
        <div class="col-12 col-md-3 bd-topbar-whitespace site-navigation show"></div>
        <div class="col pl-md-4 topbar-main">
            
            <button id="navbar-toggler" class="navbar-toggler ml-0" type="button" data-toggle="collapse"
                data-toggle="tooltip" data-placement="bottom" data-target=".site-navigation" aria-controls="navbar-menu"
                aria-expanded="true" aria-label="Toggle navigation" aria-controls="site-navigation"
                title="Toggle navigation" data-toggle="tooltip" data-placement="left">
                <i class="fas fa-bars"></i>
                <i class="fas fa-arrow-left"></i>
                <i class="fas fa-arrow-up"></i>
            </button>
            
            
<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn" aria-label="Download this page"><i
            class="fas fa-download"></i></button>

    <div class="dropdown-buttons">
        <!-- ipynb file if we had a myst markdown file -->
        
        <!-- Download raw file -->
        <a class="dropdown-buttons" href="../_sources/Ex6/proteinmodelling.md"><button type="button"
                class="btn btn-secondary topbarbtn" title="Download source file" data-toggle="tooltip"
                data-placement="left">.md</button></a>
        <!-- Download PDF via print -->
        <button type="button" id="download-print" class="btn btn-secondary topbarbtn" title="Print to PDF"
                onclick="printPdf(this)" data-toggle="tooltip" data-placement="left">.pdf</button>
    </div>
</div>

            <!-- Source interaction buttons -->

<div class="dropdown-buttons-trigger">
    <button id="dropdown-buttons-trigger" class="btn btn-secondary topbarbtn"
        aria-label="Connect with source repository"><i class="fab fa-github"></i></button>
    <div class="dropdown-buttons sourcebuttons">
        <a class="repository-button"
            href="https://github.com/lcbc-epfl/mdmc-public"><button type="button" class="btn btn-secondary topbarbtn"
                data-toggle="tooltip" data-placement="left" title="Source repository"><i
                    class="fab fa-github"></i>repository</button></a>
        <a class="issues-button"
            href="https://github.com/lcbc-epfl/mdmc-public/issues/new?title=Issue%20on%20page%20%2FEx6/proteinmodelling.html&body=Your%20issue%20content%20here."><button
                type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip" data-placement="left"
                title="Open an issue"><i class="fas fa-lightbulb"></i>open issue</button></a>
        
    </div>
</div>

            <!-- Full screen (wrap in <a> to have style consistency -->

<a class="full-screen-button"><button type="button" class="btn btn-secondary topbarbtn" data-toggle="tooltip"
        data-placement="bottom" onclick="toggleFullScreen()" aria-label="Fullscreen mode"
        title="Fullscreen mode"><i
            class="fas fa-expand"></i></button></a>

            <!-- Launch buttons -->

        </div>

        <!-- Table of contents -->
        <div class="d-none d-md-block col-md-2 bd-toc show noprint">
            
            <div class="tocsection onthispage pt-5 pb-3">
                <i class="fas fa-list"></i> Contents
            </div>
            <nav id="bd-toc-nav" aria-label="Page">
                <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#molecular-dynamics-simulations-in-biochemistry">
   6.3.1. Molecular Dynamics Simulations in Biochemistry
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#amino-acids-and-proteins">
   6.3.2. Amino Acids and Proteins
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#amino-acids">
     6.3.2.1. Amino Acids
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#proteins">
     6.3.2.2. Proteins
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#proteins-form-and-function">
       6.3.2.2.1. Proteins: Form and Function
      </a>
     </li>
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#alpha-helices-and-beta-sheets">
       6.3.2.2.2.
       <span class="math notranslate nohighlight">
        \(\alpha\)
       </span>
       -helices and
       <span class="math notranslate nohighlight">
        \(\beta\)
       </span>
       -sheets
      </a>
     </li>
    </ul>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#protein-assemblies">
   6.3.3. Protein Assemblies
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#secondary-structures">
     6.3.3.1. Secondary Structures
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#tertiary-structures">
     6.3.3.2. Tertiary Structures
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#quaternary-structures">
     6.3.3.3. Quaternary Structures
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#protein-folding">
   6.3.4. Protein Folding
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#computational-methods">
   6.3.5. Computational methods
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein">
     6.3.5.1. Analysis of the Folding Dynamics of the Trp-cage Miniprotein
    </a>
   </li>
  </ul>
 </li>
</ul>

            </nav>
        </div>
    </div>
</div>
    <div id="main-content" class="row">
        <div class="col-12 col-md-9 pl-md-3 pr-md-0">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Protein Modelling</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#molecular-dynamics-simulations-in-biochemistry">
   6.3.1. Molecular Dynamics Simulations in Biochemistry
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#amino-acids-and-proteins">
   6.3.2. Amino Acids and Proteins
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#amino-acids">
     6.3.2.1. Amino Acids
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#proteins">
     6.3.2.2. Proteins
    </a>
    <ul class="nav section-nav flex-column">
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#proteins-form-and-function">
       6.3.2.2.1. Proteins: Form and Function
      </a>
     </li>
     <li class="toc-h4 nav-item toc-entry">
      <a class="reference internal nav-link" href="#alpha-helices-and-beta-sheets">
       6.3.2.2.2.
       <span class="math notranslate nohighlight">
        \(\alpha\)
       </span>
       -helices and
       <span class="math notranslate nohighlight">
        \(\beta\)
       </span>
       -sheets
      </a>
     </li>
    </ul>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#protein-assemblies">
   6.3.3. Protein Assemblies
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#secondary-structures">
     6.3.3.1. Secondary Structures
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#tertiary-structures">
     6.3.3.2. Tertiary Structures
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#quaternary-structures">
     6.3.3.3. Quaternary Structures
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#protein-folding">
   6.3.4. Protein Folding
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#computational-methods">
   6.3.5. Computational methods
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein">
     6.3.5.1. Analysis of the Folding Dynamics of the Trp-cage Miniprotein
    </a>
   </li>
  </ul>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            
              <div>
                
  <div class="tex2jax_ignore mathjax_ignore section" id="protein-modelling">
<h1><span class="section-number">6.3. </span>Protein Modelling<a class="headerlink" href="#protein-modelling" title="Permalink to this headline">¶</a></h1>
<div class="section" id="molecular-dynamics-simulations-in-biochemistry">
<h2><span class="section-number">6.3.1. </span>Molecular Dynamics Simulations in Biochemistry<a class="headerlink" href="#molecular-dynamics-simulations-in-biochemistry" title="Permalink to this headline">¶</a></h2>
<p>Individual cellular processes are complex; involving a plethora of
reactants, reaction pathways, intermediary states and biological
catalysts. Such processes are often very large (<span class="math notranslate nohighlight">\(&gt;1,000,000\)</span> atoms), and
with modern computing power, treating such large systems at a full <em>ab
initio</em> level is not appropriate. Understanding these reaction pathways
in view of developing drugs or treatments for diseases is often an
immense undertaking. For example, the structural determination of
proteins by experiment requires that these proteins are synthesised,
crystallised and purified. One may, however, encounter difficult
transfection, suffer from low yields or high losses during purification:
all of which lead to very high monetary cost. Crystallisation techniques
often do not capture the protein of interest in its active state, and
cannot isolate some proteins at all, hence may not be of much benefit to
perform.</p>
<p>Computational modelling is generally a useful tool in cases where
mechanisms or structures are poorly understood. Additionally when using
molecular dynamics, processes can be captured explicitly, thus analysis
can be performed in visual and mathematical detail, in an environment in
which the protein is active.</p>
</div>
<div class="section" id="amino-acids-and-proteins">
<h2><span class="section-number">6.3.2. </span>Amino Acids and Proteins<a class="headerlink" href="#amino-acids-and-proteins" title="Permalink to this headline">¶</a></h2>
<div class="section" id="amino-acids">
<h3><span class="section-number">6.3.2.1. </span>Amino Acids<a class="headerlink" href="#amino-acids" title="Permalink to this headline">¶</a></h3>
<p>Amino acids are the structural units that make up proteins. They join
together via dehydration (<a class="reference internal" href="#dehydration-reaction"><span class="std std-numref">Fig. 6.1</span></a>) to form short polymer chains named peptides,
or longer chains called polypeptides or proteins. These polypeditic
chains are unbranched, with each amino acid within the chain attached to
two neighbouring amino acids. There are a total of 23 amino acids that
are naturally incorporated into polypeptides, termed proteinogenic amino
acids. Of these, 21 are encoded by the universal genetic code, while the
remaining two, selenocysteine and pyrrolysine, are incorporated into
polypeptides by unique biological mechanisms.</p>
<div class="figure align-default" id="dehydration-reaction">
<img alt="../_images/dehydration_reaction.png" src="../_images/dehydration_reaction.png" />
<p class="caption"><span class="caption-number">Fig. 6.1 </span><span class="caption-text">Dehydration of two amino acids to yield a dipeptide. Each amino acid
differs in the <strong>R</strong> group, thus polypeptide chains have a preserved and
regular backbone. Public domain <a class="reference external" href="https://commons.wikimedia.org/wiki/File:AminoacidCondensation.svg">V8rik</a></span><a class="headerlink" href="#dehydration-reaction" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="proteins">
<h3><span class="section-number">6.3.2.2. </span>Proteins<a class="headerlink" href="#proteins" title="Permalink to this headline">¶</a></h3>
<p>Proteins are large macromolecules consisting of one or more polypeptide
chains. They differ from one another primarily in their sequence of
amino acids, which is dictated by the genetic coding responsible for
that particular protein. The amino acid sequence results in a folded
protein with a specific structure that determines its activity for a
particular process. Proteins perform the majority of the vast array of
functions required for life within living organisms, including:
metabolic catalysis, DNA replication and signal transduction.</p>
<div class="section" id="proteins-form-and-function">
<h4><span class="section-number">6.3.2.2.1. </span>Proteins: Form and Function<a class="headerlink" href="#proteins-form-and-function" title="Permalink to this headline">¶</a></h4>
<p>The activity and task of a protein, in large part, is determined by its
3D structural characteristics. This is defined by the sequence of amino
acids in the polypeptide chains that constitute the protein, which, in
turn, is defined by the sequence of genetic code on the encoding mRNA
strand.</p>
</div>
<div class="section" id="alpha-helices-and-beta-sheets">
<span id="alpha-helix"></span><h4><span class="section-number">6.3.2.2.2. </span><span class="math notranslate nohighlight">\(\alpha\)</span>-helices and <span class="math notranslate nohighlight">\(\beta\)</span>-sheets<a class="headerlink" href="#alpha-helices-and-beta-sheets" title="Permalink to this headline">¶</a></h4>
<p>Polypeptide chains are not usually linear, in fact, they can adopt very
specific conformations which are vitally important to the structural
characteristics of proteins. The most common conformations are the
<span class="math notranslate nohighlight">\(\alpha\)</span>-helix and <span class="math notranslate nohighlight">\(\beta\)</span>-sheet (<a class="reference internal" href="#polypeptide-structures"><span class="std std-numref">Fig. 6.2</span></a>), however, there are variants which
differ in their symmetry. The <span class="math notranslate nohighlight">\(\alpha\)</span>-helix conformation results when
every backbone <span class="math notranslate nohighlight">\(N-H\)</span> group donates a hydrogen bond to the backbone <span class="math notranslate nohighlight">\(C=O\)</span>
of the amino acid four residues earlier (<span class="math notranslate nohighlight">\(i+4 \rightarrow i\)</span> hydrogen
bonding). Conversely, <span class="math notranslate nohighlight">\(\beta\)</span>-sheets consist of <span class="math notranslate nohighlight">\(\beta\)</span>-strands (linear
polypeptides) connected laterally by at least two or three backbone
hydrogen bonds, forming a generally twisted pleated sheet.</p>
<p>There are multiple larger-scale structures in which a number of
<span class="math notranslate nohighlight">\(\alpha\)</span>-helical or <span class="math notranslate nohighlight">\(\beta\)</span>-sheets assemble into complicated motifs.
These include: coiled-coils, <span class="math notranslate nohighlight">\(\beta\)</span>-hairpin, Greek key motif,
<span class="math notranslate nohighlight">\(\beta\)</span>-<span class="math notranslate nohighlight">\(\alpha\)</span>-<span class="math notranslate nohighlight">\(\beta\)</span> motif, <span class="math notranslate nohighlight">\(\beta\)</span>-meander motif and <span class="math notranslate nohighlight">\(\Psi\)</span>-loop
motif.</p>
 <script src="https://3Dmol.org/build/3Dmol-min.js" async></script>     
<div style="height: 400px; width: 100%; position: relative;" class='viewer_3Dmoljs' data-href='alpha_helix.pdb' data-backgroundcolor='0xffffff' ddata-select1='chain:A' data-style1='cartoon' data-select2='chain:A' data-style2='cartoon;stick'></div> <div style="height: 400px; width: 100%; position: relative;" class='viewer_3Dmoljs' data-href='beta_sheet.pdb' data-backgroundcolor='0xffffff' data-select1='chain:A' data-style1='cartoon' data-select2='chain:A' data-style2='cartoon;stick'></div>       
<div class="figure align-default" id="polypeptide-structures">
<a class="reference internal image-reference" href="../_images/empty.svg"><img alt="../_images/empty.svg" height="1px" src="../_images/empty.svg" /></a>
<p class="caption"><span class="caption-number">Fig. 6.2 </span><span class="caption-text">Hydrogen-bonding is the primary driving force of the formation of <span class="math notranslate nohighlight">\(\alpha\)</span>-helices and <span class="math notranslate nohighlight">\(\beta\)</span>-sheets. This in turn depends on the type and sequence of amino acids involved in the polypeptide. Top alpha helix from the GB1 protein, bottom beta hairpin from the GB1 protein containing a beta sheet connected with a short loop.</span><a class="headerlink" href="#polypeptide-structures" title="Permalink to this image">¶</a></p>
</div>
</div>
</div>
</div>
<div class="section" id="protein-assemblies">
<h2><span class="section-number">6.3.3. </span>Protein Assemblies<a class="headerlink" href="#protein-assemblies" title="Permalink to this headline">¶</a></h2>
<p>There are three common structures involving proteins: secondary,
tertiary and quaternary structures (<a class="reference internal" href="#protein-structures"><span class="std std-numref">Fig. 6.3</span></a>). The study of these structures is vital
to the understanding of diseases, metabolism and other biological
processes.</p>
<div class="section" id="secondary-structures">
<h3><span class="section-number">6.3.3.1. </span>Secondary Structures<a class="headerlink" href="#secondary-structures" title="Permalink to this headline">¶</a></h3>
<p>Secondary structures evolve from the interaction behaviour of hydrogen
bonding formed between the amine hydrogen and carbonyl oxygen atoms
contained in the backbone peptide bonds of the protein. There are a
number of secondary structures observed in nature, the two most common
being the <span class="math notranslate nohighlight">\(\alpha\)</span>-helix and <span class="math notranslate nohighlight">\(\beta\)</span>-sheet described in <a class="reference internal" href="#alpha-helix"><span class="std std-numref">Section 6.3.2.2.2</span></a></p>
</div>
<div class="section" id="tertiary-structures">
<h3><span class="section-number">6.3.3.2. </span>Tertiary Structures<a class="headerlink" href="#tertiary-structures" title="Permalink to this headline">¶</a></h3>
<p>Tertiary structures are proteins which have a single polypeptide chain,
with one or more protein secondary structures.</p>
</div>
<div class="section" id="quaternary-structures">
<h3><span class="section-number">6.3.3.3. </span>Quaternary Structures<a class="headerlink" href="#quaternary-structures" title="Permalink to this headline">¶</a></h3>
<p>Quaternary structures involve the folding of multiple folded protein or
coiling protein molecules in a multi-subunit complex.</p>
<div class="figure align-default" id="protein-structures">
<a class="reference internal image-reference" href="../_images/protein_structure.jpg"><img alt="../_images/protein_structure.jpg" src="../_images/protein_structure.jpg" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 6.3 </span><span class="caption-text">Depiction of the components of quaternary
structures. (CC by SA 3.0 <a class="reference external" href="http://cnx.org/content/col11496/1.6/">
OpenStax College - Anatomy &amp; Physiology,</a>)</span><a class="headerlink" href="#protein-structures" title="Permalink to this image">¶</a></p>
</div>
</div>
</div>
<div class="section" id="protein-folding">
<h2><span class="section-number">6.3.4. </span>Protein Folding<a class="headerlink" href="#protein-folding" title="Permalink to this headline">¶</a></h2>
<p>Protein folding is the process by which a protein assumes its functional
shape or conformation. Each protein exists as an unfolded polypeptide
chain when translated from a sequence of mRNA to a linear chain of amino
acids (Figure <a class="reference internal" href="#id1"><span class="std std-numref">Fig. 6.4</span></a>). Early folding already occurs in the exit tunnel of the ribosome (such as formation of alpha helices). After exiting the polypeptide
chain then further folds up in a complex process than can involve multiple intermediary state which sometimes require chaperone proteins to fold the protein into the global energy minimum. This free energy minimum exists because displacement of ordered solvent around the hydrophobic core into bulk solvent is favorable (so called hydrophobic collapse, <a class="reference internal" href="#id1"><span class="std std-numref">Fig. 6.4</span></a>).</p>
<div class="figure align-default" id="id1">
<a class="reference internal image-reference" href="../_images/Protein_folding_schematic.png"><img alt="../_images/Protein_folding_schematic.png" src="../_images/Protein_folding_schematic.png" style="height: 400px;" /></a>
<p class="caption"><span class="caption-number">Fig. 6.4 </span><span class="caption-text">An example of a protein primary chain folding into a complex tertiary
structure by hydrophobic collapse. Filled black dots are hydrophobic amino acids. Public Domain <a class="reference external" href="https://commons.wikimedia.org/wiki/File:Protein_folding_schematic.png">Tomixdf</a></span><a class="headerlink" href="#id1" title="Permalink to this image">¶</a></p>
</div>
</div>
<div class="section" id="computational-methods">
<span id="id2"></span><h2><span class="section-number">6.3.5. </span>Computational methods<a class="headerlink" href="#computational-methods" title="Permalink to this headline">¶</a></h2>
<div class="section" id="analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein">
<h3><span class="section-number">6.3.5.1. </span>Analysis of the Folding Dynamics of the Trp-cage Miniprotein<a class="headerlink" href="#analysis-of-the-folding-dynamics-of-the-trp-cage-miniprotein" title="Permalink to this headline">¶</a></h3>
<p>The Trp-cage miniprotein is a designed and stable polypeptide that
behaves like larger globular proteins. It is currently the smallest
protein to display two state folding properties which is easy to analyze in the laboratory e.g via thermal or chemical denaturation. The small size of this
protein makes it an ideal candidate for computational folding
simulations, thus it is frequently used to benchmark new computational
techniques and as a model system to compare folding morphologies in
larger systems. Simulating the folding dynamics of this protein with
explicitly defined water molecules is both a lengthy and computationally
costly process <a class="reference internal" href="#trpcagekinetics"><span class="std std-numref">Fig. 6.5</span></a>. Instead, we make use of an implicit
solvent model to reduce computational cost and still include, in
approximate detail, the effects of water solvation. This is inspired by work of Simmerling et. al.<span id="id3">[<a class="reference internal" href="../intro.html#id4" title="Carlos Simmerling, Bentley Strockbine, and Adrian E. Roitberg. All-atom structure prediction and folding simulations of a stable protein. Journal of the American Chemical Society, 124(38):11258-11259, 2002. doi:10.1021/ja0273851.">SSR02</a>]</span> in 2004.</p>
<div class="figure align-default" id="trpcagekinetics">
<img alt="../_images/trpc.png" src="../_images/trpc.png" />
<p class="caption"><span class="caption-number">Fig. 6.5 </span><span class="caption-text">A computationally determined kinetic model of Trp-cage showing different states in the folding process and their interconversion rates. Yellow unfolded protein, red folded state, gray molten globule (i.e partially folded states)  <a class="reference external" href="https://doi.org/10.1371/journal.pcbi.1000452">Marinelli et. al.</a> CC by SA</span><a class="headerlink" href="#trpcagekinetics" title="Permalink to this image">¶</a></p>
</div>
<div class="figure align-default" id="startingtrpstructure">
<img alt="../_images/startingStructure2.png" src="../_images/startingStructure2.png" />
<p class="caption"><span class="caption-number">Fig. 6.6 </span><span class="caption-text">Trp-cage starting structure in which cyan represents carbon, red is oxygen, white is hydrogen and blue is nitrogen. Built using the xLeap program within the AmberTools package.</span><a class="headerlink" href="#startingtrpstructure" title="Permalink to this image">¶</a></p>
</div>
</div>
</div>
</div>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./Ex6"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            
                <!-- Previous / next buttons -->
<div class='prev-next-area'> 
    <a class='left-prev' id="prev-link" href="theory.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title"><span class="section-number">6.1. </span>Theory</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="TRP_Cage_OpenMM_colab.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title"><span class="section-number">6.4. </span>Practical: TRP Cage using OpenMM</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            
        </div>
    </div>
    <footer class="footer">
  <p>
    
      By Simon Duerr, Sophie Johnson, Irea Mosquera, Andrea Levy,  Ursula Roethlisberger<br/>
    
      <div class="extra_footer">
        <div class="container-lg mt-3 footer"> <div class="row"> <div class="col-xl-9 d-flex"> <div class="container"> <div class="row"> <div class="col-6 d-flex" style="justify-content:center"> <a href="https://lcbc.epfl.ch"> Laboratory of Computational Biochemistry and Chemistry</a></li>
</div> <div class="col-6 d-flex" style="justify-content:center"> <ul style="list-style:none"> <li> <a href="https://github.com/lcbc-epfl/mdmc-public">GitHub </a> </li> </li> <!--<li> <a href="https://doi.org/10.5281/zenodo.0000000"><img src="https://zenodo.org/badge/DOI/10.5281/zenodo.00000000.svg" alt="DOI"></a> </li>--> </ul> </div> </div> </div> </div> <div class="col-xl-3"> <a href="https://epfl.ch"><img src="https://www.epfl.ch/wp/5.5/wp-content/themes/wp-theme-2018/assets/svg/epfl-logo.svg?refresh=now" alt="EPFL Logo" style="padding:5px;height:33px;"></a> </div> </div> </div> <div class="row_footer my-3"> Dual licensed under the <a href="https://creativecommons.org/licenses/by-sa/4.0/" target="_blank">Creative Commons Attribution Share Alike 4.0 International</a> (CC BY SA 4.0) and MIT license. (Except where indicated otherwise) </div>
      </div>
  </p>
</footer>
</main>


      </div>
    </div>
  
  <script src="../_static/js/index.be7d3bbb2ef33a8344ce.js"></script>

  </body>
</html>